### DISCUSSION TITLE
Erreur Not found

### DISCUSSION THREAD

**Discussion by Dimen7io** [LimeDrive/ygg-rss-proxy#26] [General]
Bonjour après avoir installé ygg-rss-proxy et flaresolverr une erreur not found apparait dans mon browser quand je tape l'adresse avec le localhost et l'url de rss avec mon passkey .
Voici ce que j'obtiens dans mon terminal

> C:\Users\Gally\Downloads>docker-compose up
time="2024-06-25T14:25:32+02:00" level=warning msg="C:\\Users\\Gally\\Downloads\\docker-compose.yml: `version` is obsolete"
[+] Running 3/0
 ✔ Container flaresolverr   Created                                                                                0.0s
 ✔ Container qbittorrent    Created                                                                                0.0s
 ✔ Container ygg-rss-proxy  Created                                                                                0.0s
Attaching to flaresolverr, qbittorrent, ygg-rss-proxy
flaresolverr   | 2024-06-25 14:25:32 INFO     FlareSolverr 3.3.20
flaresolverr   | 2024-06-25 14:25:32 INFO     Testing web browser installation...
flaresolverr   | 2024-06-25 14:25:32 INFO     Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.31
flaresolverr   | 2024-06-25 14:25:32 INFO     Chrome / Chromium path: /usr/bin/chromium
flaresolverr   | 2024-06-25 14:25:32 INFO     Chrome / Chromium major version: 120
flaresolverr   | 2024-06-25 14:25:32 INFO     Launching web browser...
qbittorrent    | [migrations] started
qbittorrent    | [migrations] no migrations found
qbittorrent    | usermod: no changes
qbittorrent    | ───────────────────────────────────────
qbittorrent    |
qbittorrent    |       ██╗     ███████╗██╗ ██████╗
qbittorrent    |       ██║     ██╔════╝██║██╔═══██╗
qbittorrent    |       ██║     ███████╗██║██║   ██║
qbittorrent    |       ██║     ╚════██║██║██║   ██║
qbittorrent    |       ███████╗███████║██║╚██████╔╝
qbittorrent    |       ╚══════╝╚══════╝╚═╝ ╚═════╝
qbittorrent    |
qbittorrent    |    Brought to you by linuxserver.io
qbittorrent    | ───────────────────────────────────────
qbittorrent    |
qbittorrent    | To support LSIO projects visit:
qbittorrent    | https://www.linuxserver.io/donate/
qbittorrent    |
qbittorrent    | ───────────────────────────────────────
qbittorrent    | GID/UID
qbittorrent    | ───────────────────────────────────────
qbittorrent    |
qbittorrent    | User UID:    1000
qbittorrent    | User GID:    1000
qbittorrent    | ───────────────────────────────────────
qbittorrent    | Linuxserver.io version: 4.6.5-r0-ls336
qbittorrent    | Build-date: 2024-06-23T06:52:30+00:00
qbittorrent    | ───────────────────────────────────────
qbittorrent    |
qbittorrent    | [custom-init] No custom files found, skipping...
qbittorrent    | WebUI will be started shortly after internal preparations. Please wait...
ygg-rss-proxy  | /usr/local/lib/python3.11/site-packages/pydantic_settings/sources.py:431: UserWarning: directory "/run/secrets" does not exist
ygg-rss-proxy  |   warnings.warn(f'directory "{self.secrets_path}" does not exist')
qbittorrent    |
qbittorrent    | ******** Information ********
qbittorrent    | To control qBittorrent, access the WebUI at: http://localhost:8080
qbittorrent    |
qbittorrent    | The WebUI administrator username is: admin
qbittorrent    | The WebUI administrator password was not set. A temporary password is provided for this session: aNBwA2Rtz
qbittorrent    | You should set your own password in program preferences.
ygg-rss-proxy  | 2024-06-25T14:25:32.911280+0200 INFO <module> ----------------------------------------------------------
ygg-rss-proxy  | 2024-06-25T14:25:32.911695+0200 INFO <module> ygg_rss_proxy version: v1.3.3-alpha
ygg-rss-proxy  | 2024-06-25T14:25:32.911925+0200 INFO <module> ----------------------------------------------------------
ygg-rss-proxy  | 2024-06-25T14:25:32.912023+0200 INFO <module> Checking directories...
ygg-rss-proxy  | 2024-06-25T14:25:32.912188+0200 INFO <module> ----------------------------------------------------------
ygg-rss-proxy  | 2024-06-25T14:25:32.912519+0200 INFO <module> Directory /app/config already exists.
ygg-rss-proxy  | 2024-06-25T14:25:32.912737+0200 INFO <module> Directory /app/config already exists.
ygg-rss-proxy  | 2024-06-25T14:25:32.912954+0200 INFO <module> Directory /app/config/logs already exists.
ygg-rss-proxy  | 2024-06-25T14:25:32.913081+0200 INFO <module> ----------------------------------------------------------
ygg-rss-proxy  | 2024-06-25T14:25:32.913236+0200 INFO <module> Starting ygg_rss_proxy on http://ygg-rss-proxy:8080
ygg-rss-proxy  | 2024-06-25T14:25:32.913310+0200 INFO <module> ----------------------------------------------------------
ygg-rss-proxy  | [2024-06-25 14:25:32 +0200] [1] [INFO] Starting gunicorn 22.0.0
ygg-rss-proxy  | [2024-06-25 14:25:32 +0200] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
ygg-rss-proxy  | [2024-06-25 14:25:32 +0200] [1] [INFO] Using worker: sync
ygg-rss-proxy  | [2024-06-25 14:25:32 +0200] [8] [INFO] Booting worker with pid: 8
ygg-rss-proxy  | [2024-06-25 14:25:33 +0200] [9] [INFO] Booting worker with pid: 9
ygg-rss-proxy  | [2024-06-25 14:25:33 +0200] [10] [INFO] Booting worker with pid: 10
flaresolverr   | 2024-06-25 14:25:33 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
flaresolverr   | 2024-06-25 14:25:33 INFO     Test successful!
flaresolverr   | 2024-06-25 14:25:33 INFO     Serving on http://0.0.0.0:8191
ygg-rss-proxy  | [2024-06-25 14:25:33 +0200] [11] [INFO] Booting worker with pid: 11
qbittorrent    | Connection to localhost (::1) 8080 port [tcp/http-alt] succeeded!
qbittorrent    | [ls.io-init] done.<

**Comment by LimeDrive** [comment]
Bonjour, par défaut ygg_rss_proxy n'est pas exposé en dehors du résaux docker sur le docker-compose.yml fourni sur le dépôt qui inclut qbit.

Dans votre qbit ajouté le flux RSS. Non pas sur localhost mais avec le noms du container : http://ygg-rss-proxy:8080/RSS?( Rest des params de l'URL RSS )

Étant dans le même résaux le container qbit vas résoudre l'adresse sans problèmes.

**Comment by LimeDrive** [comment]
Le problème étant dans la config et pas dans le code, je transforme cette issues en discussion.
